include/cpp/jank/runtime/behavior/associatively_writable.hpp

```diff
@@ -14,7 +14,7 @@ namespace jank::runtime::behavior
   template <typename T>
   concept associatively_writable_in_place = requires(T * const t) {
     {
-      t->assoc_in_place(object_ptr{}, object_ptr{})
+      t->assoc_in_place(object_ptr{}, object_ptr{}), t->dissoc_in_place(object_ptr{})
```

Comment:
This needs to have its own entry. Concepts here, with the `requires` expression, are saying `<expression> -> <type>`, meaning `<this expression> results in <some type>`. For an example of this, you can look at `seqable.hpp`.